gradle/plugins/src/main/kotlin/com.hedera.gradle.application.gradle.kts

```diff
@@ -1,3 +1,27 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import gradle.kotlin.dsl.accessors._3dd858cc73bb8a89777520d17e15f078.assemble
+import gradle.kotlin.dsl.accessors._3dd858cc73bb8a89777520d17e15f078.jar
+import gradle.kotlin.dsl.accessors._3dd858cc73bb8a89777520d17e15f078.runtimeClasspath
```

Comment:
You need to delete these imports. If Intellij doesnt compile you can do `./gradlew --stop` and `/gradlew assemble --rerun-tasks`

Reply:
I removed them. IntelliJ compiles.